src/test/javascript/spec/component/modeling-exercise/modeling-exercise.component.spec.ts

```diff
@@ -138,27 +140,30 @@ describe('ModelingExercise Management Component', () => {
     it('should return items id when tracked', () => {
         const item = new ModelingExercise(UMLDiagramType.ClassDiagram, undefined, undefined);
         item.id = 123;
-        expect(comp.trackId(2, item)).toEqual(123);
+        expect(comp.trackId(2, item)).toBe(123);
     });
 
     it('should delete the given exercise', fakeAsync(() => {
-        const deleteStub = jest.spyOn(modelingExerciseService, 'delete').mockReturnValue(of({} as HttpResponse<{}>));
-        const broadcastStub = jest.spyOn(eventManager, 'broadcast');
+        const deleteSpy = jest.spyOn(modelingExerciseService, 'delete').mockReturnValue(of({} as HttpResponse<{}>));
```

Comment:
This is not a spy though. It returns the same default value for all calls. Therefore "Stub" is correct